schemes/musica/musica_ccpp_namelist.xml

```diff
@@ -75,16 +75,20 @@
      units
           This is the CCPP unit specification of the variable (e.g., m s-1).
                                                                         -->
-  <section name="MUSICA Standard Names">
-    <standard_name name="micm_solver_type">
-      <type kind="" units="none">integer</type>
-    </standard_name>
-    <standard_name name="number_of_grid_cells">
-      <type kind="" units="count">integer</type>
-    </standard_name>
-  </section>
-
 <!-- MUSICA configuration options -->
+  <entry id="micm_solver_type">
+    <type>integer</type>
+    <category>musica_ccpp</category>
+    <group>musica_ccpp</group>
+    <standard_name>micm_solver_type</standard_name>
+    <units>none</units>
+    <desc>
+      The type of MICM solver to use.
+    </desc>
+    <values>
+      <value>1</value>
+    </values>
+  </entry>
```

Comment:
I’m hesitant about setting the solver type using an integer, as this implies that users need to know the enum type and which solver corresponds to each integer. Can I ask why we want to include the solver type in the xml?

Reply:
After I went to try to run the MUSICA scheme in CAM-SIMA, I realized that we have several inputs to MUSICA (like the solver type) that need to be provided from "somewhere". The options (at least that I'm aware of) were to put this in this xml file, or to make it available from CAM-SIMA, like the solar zenith angle, etc. I didn't seem like a good idea to me to put this in CAM-SIMA, so I put it here. But, maybe @nusbaume or @peverwhee might have a better suggestion for where to put this?

Also, I bet we could change this to a string and convert it to the enum in the fortran code. Would that be better do you think?

Reply:
I see. I think I would do the same and put the solver type in this xml file. I like the idea of using a string and converting it to an integer in the code, though I’m not sure if there’s a way to do this in a simple way in Fortran. But this explanation makes sense to me, and I don’t mind this implementation.

Reply:
I think the location you have now (the namelist XML file) makes a lot of sense, so I would personally vote to just leave it there. Thanks!